client-lib/go/client/jvs_config.go

```diff
@@ -37,6 +37,9 @@ type JVSConfig struct {
 
 	// CacheTimeout is the duration that keys stay in cache before being revoked.
 	CacheTimeout time.Duration `yaml:"cache_timeout" env:"CACHE_TIMEOUT,overwrite,default=5m"`
+
+	// ForbidBreakglass represents whether the jvs client forbids breakglass.
+	ForbidBreakglass bool `yaml:"forbid_breakglass" env:"FORBID_BREAKGLASS,overwrite,default=false"`
```

Comment:
I think the default should be to forbid breakglass. Users should have to opt-in to allowing breakglass. As such, we should rename this to `AllowBreakglass`, but keep the default as false. @yolocs

Reply:
To me, breakglass is a use case most services should and will adopt. That's why I think it should be allowed by default. Is your preference from being more strict by default?

Reply:
If it's common for services to allow breakglass, I think it's fine to make the default to be allow breakglass. Otherwise, it's safer to be more strict by default. @sethvargo Any ideas?

Reply:
Breakglass is inherently a way to bypass security controls and therefore should be disabled by default. As a rule of thumb, users should always have to opt into doing something insecure, even if it's a standard.

Reply:
done. Defaults to breakglass forbidden.

Reply:
(I don't think JVS is technically a security control. Authz is security control. JVS is privacy focused.)

The risk on the other side seems to be an oncaller requesting access at 3am only ends up finding out the service doesn't allow breakglass due to it being the default.
I can't say which risk is worse, especially when we will still audit log the breakglass access. Unable to debug in worse cases could stop business continuity.

I'm fine flipping the flag if you think the security risk outweighs the risk on the other side.